src/locales/en/translation.json

```diff
@@ -162,6 +162,13 @@
           "title": "Query Timeout",
           "description": "Terminates the query if no search results are returned within this time. Default: 500ms."
         }
+      },
+      "appearance": {
+        "title": "Appearance Settings",
+        "opacity": {
+          "title": "Transparency When Pinned and Unfocused",
```

Comment:
Title: Pinned Window Dimness Setting

Description: Adjusts the opacity level of the Coco AI window when it’s pinned and not in focus. Set a value between 10% and 100%, where 100% means fully opaque (no dimming), and lower values increase transparency, allowing underlying content to show through.